create_symlinks.sh

```diff
@@ -9,5 +9,5 @@ destination=$1
 find $1 -maxdepth 1 -type l -exec rm -f {} \;
 
 # Link all the things
-ln -s `pwd`/*.lic $1
+ln -sf `pwd`/*.lic $1
 ln -s `pwd`/profiles/ $1
```

Comment:
I symlink my profiles (from a separate directory) into dr-scripts/profiles after running this script. Not sure if thats the best way to do handle it, but it seems to work